blockchain/blockchain.go

```diff
@@ -68,7 +68,7 @@ const (
 )
 
 const (
-	triesInMemory = 4
+	triesInMemory = 128
```

Comment:
This change will cause a performance degradation. If you changed this to make `traceBlock` and `traceChain` work, please consider applying this only if it traces a block or chain.